tests/accel/internal/accelconfig/config.go

```diff
@@ -0,0 +1,40 @@
+package accelconfig
+
+import (
+	"log"
+
+	"github.com/golang/glog"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	"github.com/openshift-kni/eco-gotests/tests/internal/config"
+)
+
+// AccelConfig contains environment information related to ocp upgrade tests.
+type AccelConfig struct {
+	PullSecret           string `envconfig:"ECO_ACCEL_PULL_SECRET"`
+	Registry             string `envconfig:"ECO_ACCEL_REGISTRY"`
+	UpgradeTargetVersion string `envconfig:"ECO_ACCEL_UPGRADE_TARGET_IMAGE"`
+	HubKubeConfig        string `envconfig:"ECO_ACCEL_HUB_KUBECONFIG"`
```

Comment:
HubKubeConfig is always default KUBECONFIG variable. If you need secondary connection to Spoke please add SpokeKubeConfig instead.